optuna/samplers/tpe/sampler.py

```diff
@@ -266,7 +266,7 @@ def _sample_from_gmm(
         while samples.size < n_samples:
             active = np.argmax(self.rng.multinomial(1, weights))
             draw = self.rng.normal(loc=mus[active], scale=sigmas[active])
-            if low <= draw <= high:
```

Comment:
`high` should be exclusive (see the documentation of [Trial.suggest_uniform](https://optuna.readthedocs.io/en/stable/reference/trial.html#optuna.trial.Trial.suggest_uniform)).